aya/src/bpf.rs

```diff
@@ -233,8 +233,8 @@ impl<'a> BpfLoader<'a> {
     /// use aya::BpfLoader;
```

Comment:
Here before the # Example section we should add

```
If the `must_exist` argument is `true`, [`BpfLoader::load`] will fail with [`ParseError::SymbolNotFound`] if the loaded object code does not contain the variable.
```

You may need to tweak the rustdoc refs, one day I'll learn how they work